src/app/core/services/loader.service.ts

```diff
@@ -6,24 +6,11 @@ import { Subject } from 'rxjs';
 })
 
 export class LoaderService {
-  private _loading = false;
-  loadingStatus = new Subject();
-  loadingState = this.loadingStatus.asObservable();
-
-  get loading() {
-    return this._loading;
-  }
-
-  set loading(value) {
-    this._loading = value;
-    this.loadingStatus.next(value);
-  }
-
   startLoading() {
-    this.loading = true;
+    document.getElementById('spinner').style.display = 'block';
```

Comment:
please cache this element as the component property, to reduce DOM querying each time

rename the method to displayLoader, hideLoader respectively

Reply:
instead of display block use empty string ''. In the case, the default one (specified in css) will be applied